Mailer/app/api/v1/Handlers/Users/SubscribeHandler.php

```diff
@@ -33,26 +34,21 @@ public function __construct(
         $this->listVariantsRepository = $listVariantsRepository;
     }
 
-    public function params()
+    public function handle($payload)
     {
-        return [
-            new InputParam(InputParam::TYPE_POST_RAW, 'raw')
-        ];
-    }
+        $request = file_get_contents("php://input");
```

Comment:
Actually, the original implementation is much cleaner than reading it from `php://input`. Why did you dump it? If there wasn't any reason for it, please use it also in other API calls.

Reply:
Because I used this file as an example:
https://github.com/remp2020/remp/blob/e97ef9053cb9e28efc94d0f440178107263876a0/Mailer/app/api/v1/Handlers/Mailers/SendEmailHandler.php#L48

Reply:
So should I change it everywhere? I found it simpler because it is just one line and there is no need for params method.

Reply:
Yes please. Handlers should be shielded from globals wherever possible so we can easily test them in the future and provide custom request payloads.